### PR TITLE
3D moisture plume test case

### PR DESCRIPTION
**Pull request by matt-frey**

### CODE REVIEW

File: src/utils/phys_constants.f90 [hunk 1]
```diff
@@ -9,10 +9,10 @@ module phys_constants
     double precision, parameter :: gravity = 9.81d0
 
     ![J/kg] latent heat of condensation
-    double precision, parameter :: L_c = 2.5e6
+    double precision, parameter :: L_c = 2.501e6
```
Comment (sjboeing) [inline src/utils/phys_constants.f90:12]: L_v is the more common naming (for vaporisation)

File: models/3d/moist_3d.f90 [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            allocate(humg(0:nz, 0:ny-1, 0:nx-1))
+
+            buoyg = zero
+            humg = zero
+
+            do i = 0, nx-1
+                do j = 0, ny-1
+                    do k = 0, nz
+
+                        pos = origin + dx * dble((/i, j, k/))
```
Comment (sjboeing) [inline models/3d/moist_3d.f90:113]: Just a note that the origin needs to be at the surface for this case to work

Comment (sjboeing) [reply models/3d/moist_3d.f90:113]: As well as in the centre of the domain (we can generalise this later)

File: src/utils/phys_constants.f90 (hunk 1 above)
Comment (matt-frey) [reply src/utils/phys_constants.f90:12]: Ok!

File: models/3d/moist_3d.f90 (hunk 2 above)
Comment (matt-frey) [reply models/3d/moist_3d.f90:113]: Ok.

**Review by sjboeing — APPROVED** [review]
Looks good to me. I will try to run it and check if the output looks plausible

**Pull request merged**